source/tools/kubernetes/dask-helm-chart.md

```diff
@@ -1,2 +1,115 @@
 Dask Helm Chart
 ===============
+
+This guide provides an entry point to deploy RAPIDS into Kubernetes cluster via helm chart,
+using [dask/helm-chart](https://github.com/dask/helm-chart) as the starting point.
+This guide will walk through steps to set up a scalable rapids cluster,
+demonstrating GPU accelerated notebook workflows and scaling up and down the cluster.
+## From Dask helm chart to RAPIDS helm chart
+
+Built on top of dask-helm-chart,
+`rapids-config.yaml` file contains additional configurations required to setup RAPIDS environment.
+
+```{literalinclude} ./dask-helm-chart/rapids-config.yaml
+:language: yaml
+```
```

Comment:
Been a bit hesitant to remove the table here. But to strike a balance of note having to dual hosting the same config and user visibility, I opted to just including the file here.

Reply:
After all, yaml is designed to be human readable :)